packages/twenty-server/src/modules/workspace-member/standard-objects/workspace-member.workspace-entity.ts

```diff
@@ -94,6 +107,82 @@ export class WorkspaceMemberWorkspaceEntity extends BaseWorkspaceEntity {
   })
   userId: string;
 
+  @WorkspaceField({
+    standardId: WORKSPACE_MEMBER_STANDARD_FIELD_IDS.timeZone,
+    type: FieldMetadataType.TEXT,
+    label: 'Time zone',
+    defaultValue: "'system'",
+    description: 'User time zone',
+    icon: 'IconTimezone',
+  })
+  timeZone: string;
+
+  @WorkspaceField({
+    standardId: WORKSPACE_MEMBER_STANDARD_FIELD_IDS.dateFormat,
+    type: FieldMetadataType.SELECT,
+    label: 'Date format',
+    description: "User's preferred date format",
+    icon: 'IconCalendarEvent',
+    options: [
+      {
+        value: WorkspaceMemberDateFormatEnum.SYSTEM,
+        label: 'System',
+        position: 0,
+        color: 'turquoise',
+      },
+      {
+        value: WorkspaceMemberDateFormatEnum.MMM_D_YYYY,
+        label: 'Month First',
+        position: 1,
+        color: 'red',
+      },
+      {
+        value: WorkspaceMemberDateFormatEnum.D_MMM_YYYY,
+        label: 'Day First',
+        position: 2,
+        color: 'purple',
+      },
+      {
+        value: WorkspaceMemberDateFormatEnum.YYYY_MMM_D,
+        label: 'Year First',
+        position: 3,
+        color: 'sky',
+      },
+    ],
+    defaultValue: `'${WorkspaceMemberDateFormatEnum.SYSTEM}'`,
+  })
+  dateFormat: string;
+
+  @WorkspaceField({
+    standardId: WORKSPACE_MEMBER_STANDARD_FIELD_IDS.timeFormat,
+    type: FieldMetadataType.SELECT,
+    label: 'Time format',
+    description: "User's preferred time format",
+    icon: 'IconClock2',
+    options: [
+      {
+        value: WorkspaceMemberTimeFormatEnum.SYSTEM,
+        label: 'System',
+        position: 0,
+        color: 'sky',
+      },
+      {
+        value: WorkspaceMemberTimeFormatEnum.HH_MM,
+        label: 'Military',
```

Comment:
I think military means XXYY without separator and beginning with 0. Let's call it 24HRS / 12HRS or something that would speak more to a reader